### PR TITLE
MISC: Add warning to avoid inputs

### PR DESCRIPTION
**Pull request by nielsboecker**
This PR will add a warning in the UI about missing input validation, and a more detailed explanation in the project README.

image

### CODE REVIEW

File: HoloRepositoryUI/client/src/components/app/new_hologram/generate/ImagingStudySelectionStep.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import { IChoiceGroupOption, IDropdownOption } from "office-ui-fabric-react";
+import { IChoiceGroupOption, IDropdownOption, MessageBar, MessageBarType } from "office-ui-fabric-react";
 import { Col, Divider, Row } from "antd";
 import ImagingStudyDetailsCard from "./ImagingStudyDetailsCard";
 import { PropsWithContext, withAppContext } from "../../../shared/AppState";
 import ImagingStudySelectionInput from "./inputs/ImagingStudySelectionInput";
 import PatientForImagingStudySelectionInput from "./inputs/PatientForImagingStudySelectionInput";
 
-export interface IImagingStudySelectionStepProps extends PropsWithContext {}
+export interface IImagingStudySelectionStepProps extends PropsWithContext {
```
Comment (boonwj) [inline HoloRepositoryUI/client/src/components/app/new_hologram/generate/ImagingStudySelectionStep.tsx]: Was this change indended? Just wondering if it's a style guide recommendation.

File: HoloRepositoryUI/client/src/components/app/new_hologram/generate/ImagingStudySelectionStep.tsx [hunk 2]
```diff
@@ -18,10 +19,8 @@ export interface IExtendedChoiceGroupOption extends IChoiceGroupOption {
   endpoint: string;
 }
 
-class ImagingStudySelectionStep extends Component<
-  IImagingStudySelectionStepProps,
-  IImagingStudySelectionStepState
-> {
+class ImagingStudySelectionStep extends Component<IImagingStudySelectionStepProps,
```
Comment (boonwj) [inline HoloRepositoryUI/client/src/components/app/new_hologram/generate/ImagingStudySelectionStep.tsx]: Same for this.

**Review by boonwj — APPROVED** [review]
Sweet turnaround time after the feedback

**Comment by boonwj** [comment]
Oh yeah, there are no issues with my testing. Just merge the branch if you verify that the highlighted code changes were intended.

File: HoloRepositoryUI/client/src/components/app/new_hologram/generate/ImagingStudySelectionStep.tsx (hunk 1 above)
Comment (nielsboecker) [reply HoloRepositoryUI/client/src/components/app/new_hologram/generate/ImagingStudySelectionStep.tsx]: No. It's auto-formatted by WebStorm. It used to be synced with the `.prettierrc` config but seems to be out of sync now. I also had to disble the TS auto-formatting temporarily cause it doesn't go togehter with the Python/black pre-commit hooks. Anyway, I'll revert the formatting change.

**Pull request merged**